notebooks/community/experiments/vertex_ai_model_experimentation.ipynb

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
<p>maybe I am missing something. Shouldn't the code for tracking params in the experiment be here?</p><p> </p><p>What about creating Dataset artifacts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>Let me fix the dataset. About the parameters they are already tracked at that point. </p>

Reply:
done